### PR TITLE
docs: update PQ docs with version for GA config

### PR DESCRIPTION
**Pull request by Meschreiber**
This PR:

- Calls out the versions that support the `preview_persisted_queries` vs. `persisted_queries` config.
- Cleans up terminology ("preregistered" -> "registered", etc.)

Do not merge until PQs are in GA.

### CODE REVIEW

File: docs/source/configuration/persisted-queries.mdx [hunk 1]
```diff
@@ -40,52 +38,54 @@ For more information on other configuration aspects, see the [GraphOS persisted
 
 The router provides four configuration options that you can combine to create the recommended [security levels](#router-security-levels). This section details each configuration option. Refer to the [security levels](#router-security-levels) section for recommended combinations.
 
-#### `preview_persisted_queries`
+> From version `1.25.0` to `1.xx.0` the `persisted_queries` configuration option was called `preview_persisted_queries`. Upgrade your router to version `1.xx.0` or later to use the [generally available](/resources/product-launch-stages/#general-availability) version of the feature and the example configuration snippets below.
```
Comment (Meschreiber) [inline docs/source/configuration/persisted-queries.mdx:41]: @abernix , would you let me know what router version this ends up being released in please?

**Review by glasser — APPROVED** [review]
The diff in this PR looks great but the title and description look off? We've already merged the change to move it to GA; this is other stuff.

**Review by shorgi — APPROVED** [review]
LGTM

**Comment by Meschreiber** [comment]
Closing in favor of https://github.com/apollographql/router/pull/3963

**Pull request closed without merging**